docs/how-to/add-edit-workbook.md

```diff
@@ -38,7 +38,7 @@ While viewing an open workbook instance:
 Each workbook is stored as a serialized JSON string value in
 `deployments/workbook-data.json`.
 
-The serialized workbook data will be referenced in `azuredeploy.json` using the
+The serialized workbook data will be referenced in `azuredeploy.json` or `azuredeploy.bicep` using the
```

Comment:
why keeping both. Cant we just delete azuredeploy.json ?

Reply:
will remove in next pr